### ISSUE TITLE
Review exported PDFs from "MHV Rx" team

### ISSUE THREAD

**Issue by vmenshutin-bylight** [department-of-veterans-affairs/va.gov-team#66799]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [x] Something else 
 
 *Give a brief description of what your team needs support with.*

The "MHV Rx" team is making a new PDF template for Medications List and Medication Details pages. We need to review these samples to make sure the a11y requirements are being met as development proceeds. 

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [ ] Yes
  - [x] No

  
## Supporting artifacts
*Please provide supporting artifacts as available.*

There are sample PDFs:

[Non-VA-medications-details-Harry-Potter-10-03-2023_120911p.m..pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/12794630/Non-VA-medications-details-Harry-Potter-10-03-2023_120911p.m.pdf)
[VA-medications-details-Harry-Potter-10-03-2023_120856p.m..pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/12794631/VA-medications-details-Harry-Potter-10-03-2023_120856p.m.pdf)
[VA-medications-list-Harry-Potter-10-03-2023_120617p.m..pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/12794632/VA-medications-list-Harry-Potter-10-03-2023_120617p.m.pdf)

## Will this work be going through the Collaboration Cycle?
- [x] Yes  (eventually)
- [ ] No

## When does this work need to be done?
- Estimated launch date:
- Estimated staging review date:
- Content, Accessibility, and IA work needed by:  asap as developers are actively working

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

-	Team name: vfs-mhv-medications
-	product owner: Raquel Robertson
-	Product manager: Tanner Bird
-	Designer: @coultonbunney-usds 
-	FE engineer: @acrollet @vmenshutin-bylight 
-	Product/team Slack channel: #[mhv-medications-rx]

**Comment by SarahKay8** [comment]
**Testing device: Acrobat Adobe** 

###  PDF review - Calamine lotion (First Doc)
**Non-VA-medications-details-Harry-Potter-10-03-2023_120911p.m..pdf**

##

- Accessibility Checker Fail - Tab order failed - No interactive elements in doc, tab order won't really make a difference but it is failing accessibility checker. Not sure if possible to set in source software
- Empty artifact tags in tag tree before each **`< H3 >`** of page. 
- Empty **`< H4 >`** tags after each **`< H3 >`** of page. Please delete these in tags tree. 
- Due to the lists taking on form of Definition lists, content in **`< P >`** tag underneath **“Non-VA medications Include These Types:**” needs to be made into a "**List"**  Steps: Convert to a "**`< L >`**" Add and **`< LI >`** as a parent element inside **`< L >`** tag. Add children tags making  “*” a **`< Lbl >`** and “prescriptions you filled…” in an **`< Lbody >`** tag etc... _Page 1_ 
[Screenshot for reference](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd3NVQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c1210645eafaa4612507b3092cac38fc695066a1/4FFBC7FE-CA5A-49F0-96F7-8FFE8E26DD8C_4_5005_c.jpeg)











### PDF Review - Lactated Ringers (Second Doc) 
 **VA-medications-details-Harry-Potter-10-03-2023_120856p.m..pdf**

- Failed tab order 
- Empty **`< artifact >`** tag before **`< H1 >`**
- Empty **`< H4 >`** tags after all **`< H3 >`**'s need to be removed 
- Empty **`< artifact >`** tags after all **`< H3 >`**'s need to be removed
- Empty **`< artifact >`** tag needs to be removed after “quantity 1: “ **`< P >`** tag. _Page 1_
 


### PDF Review Medications - Lactated Ringers (Third Doc) 
**VA-medications-list-Harry-Potter-10-03-2023_120617p.m..pdf**

- Tab order failed 
- Empty artifact tags before all **`< H3 >`**s (delete In tags tree) _pages 1-7_
- Empty artifact tags before all **`< H4 >`**'s (delete in tags tree) _page 8-9_
- First bulleted point section underneath **“We may use this status for either of these reasons:”** needs to be broken out of  
 **`< P >`** tag and formatted into a **List** _Page 10_ 
- Bulleted point section under **“Nefazodone 100MG Tab”** needs to be made into a **List** _Page 10_
[Screenshot for reference](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeFFVQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--8a4e0c2edd7a43c21daa4bda1889604c25d7377f/DB4C2E74-B6F1-4E06-BE5A-456DF41A1E2D_4_5005_c.jpeg)

- Empty **`< artifact > `**tags before **`< H3 >`** need to be removed from tags tree. _Pages 10-12_


**Allergies section** 

- Empty **`< H4 >`** tag after all **`< H3 >`**’s needs to be removed from tags tree _Pages 13-17_
- Empty **`< artifact >`** tags after all **`< H3 >`**s needs to be removed from tags tree _Pages 13-17_


### Overall Feedback on all PDF(s)

- Provide content for Subject fields in **Metadata**
- **`< Sect >`** tags are used as root elements. **`< Document >`** should be used as the tag that contains all tags in tag tree.
- Overall **Heading Hierarchy** is good to go


### Header/Footer Issue to be aware of 

- Tagged **`< artifact >`** header and footer content at end of tag tree is read by screenreaders when users enter end of document. If it isn't possible to progammatically ignore header/footer in source, could alt text be added to properties of the artifacted content to let users know it should be ignored? 

 It would be best to find a way to make sure header/footer content isn't read at all. If this is content is important and should be read, only one instance of the header/footer should be accessed - the header on the the first page and the footer on the last page. This is the ideal scenario. It seems like the app is trying to make the header/footer content hidden from users due to the <Artifact> tags, but it's still picked up. Since they come at the end of the document, it doesn't impact the user much, so I'd be okay leaving as is if there's no way to address the issue.


cc: @vmenshutin-bylight @BobbyBaileyRB @artsymartha68 

If there is any thing I missed or if I need to clarify anything, lmk

**Comment by acrollet** [comment]
Regarding tab order, we're waiting on a patch to get merged to the upstream package: https://github.com/foliojs/pdfkit/pull/1449

**Comment by SarahKay8** [comment]
@acrollet Closing as a Jira Ticket has been created. [Slack thread here](https://dsva.slack.com/archives/C8E985R32/p1696604373361759)
